src/h3lib/lib/polyfill.c

```diff
@@ -418,6 +418,12 @@ void iterStepPolygonCompact(IterCellsPolygonCompact *iter) {
         iter->_started = true;
     }
 
+    if (iter->_polygon->geoloop.numVerts == 0) {
```

Comment:
Should this be changed to 0 or 1 returns nothing? @nrabinowitz

Reply:
I think the case for 1 and 2 vertexes is equivalent - in both cases, we have input that cannot be a 2D polygon, and we either have to discard it or treat it like a shape. While it's obviously a pathological case, I think it would be reasonable to assume that If you have a one-vertex "polygon" and you use `CONTAINMENT_OVERLAPPING` that you'd get the cell that contains the point. There's also a potential use case for line segments in the 2-vertex case. I'm also concerned that the estimate and the implementation don't line up.

Should I take a shot at a PR to fix this, which we can land before this one?

Reply:
I agree that seems like the expected behavior, if you have an idea for addressing please take a look.

I can help apply these tests to another branch to help validate.